aiocogeo/tiler.py

```diff
@@ -96,41 +130,46 @@ async def _warped_read(
         resample_method: int = Image.NEAREST,
     ) -> np.ndarray:
         src_transform = from_bounds(*bounds, width=width, height=height)
-        bounds = transform_bounds(bounds_crs, CRS.from_epsg(self.cog.epsg), *bounds)
+        bounds = transform_bounds(bounds_crs, CRS.from_epsg(self.epsg), *bounds)
         dst_transform = from_bounds(*bounds, width=width, height=height)
-        arr = await self.cog.read(
+        arr = await self.read(
             bounds, shape=(width, height), resample_method=resample_method
         )
         arr, _ = reproject(
             arr,
             destination=np.empty((self.profile["count"], width, height)),
             src_transform=dst_transform,
             dst_transform=src_transform,
-            src_crs=CRS.from_epsg(self.cog.epsg),
+            src_crs=CRS.from_epsg(self.epsg),
             dst_crs=bounds_crs,
         )
         return arr.astype(self.profile["dtype"])
 
-    async def info(self) -> COGInfo:
-        if self.cog.has_alpha:
+    async def info(self) -> Dict:
+        if self.has_alpha:
             nodata_type = "Alpha"
-        elif self.cog.is_masked:
+        elif self.is_masked:
             nodata_type = "Mask"
-        elif self.cog.nodata is not None:
+        elif self.nodata is not None:
             nodata_type = "Nodata"
         else:
             nodata_type = "None"
 
         # TODO: Figure out where scale, offset, band_metadata, and band_descriptions come from
-        return COGInfo(
+        band_descr = [(ix, f"band{ix}") for ix in self.indexes]
+        band_meta = [(ix, {}) for ix in self.indexes]
+
+        return dict(
```

Comment:
add to remove COGInfo because in AsyncBaseReader we assign new values (statistics) to the result of info()